src/chrome/content/rules/Ethical_Data.xml

```diff
@@ -0,0 +1,10 @@
+<ruleset name="Ethical Data">
+
+	<target host="ethicaldata.co.uk" />
+	<target host="*.ethicaldata.co.uk" />
```

Comment:
Aha, that makes sense then. Might be worth adding a comment to this end, but it's not a big deal